osu.Game/osu.Game.csproj

```diff
@@ -7,7 +7,9 @@
   <PropertyGroup Label="Nuget">
     <Title>osu!</Title>
     <PackageId>ppy.osu.Game</PackageId>
+    <Version>0.0.0</Version>
     <PackageIcon>icon.png</PackageIcon>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Not sure we want this, as it will add overhead for normal development.

Reply:
This can help finding packing warnings at normal development, for example, depending on preview package from stable.
Packing is relatively faster than building. `dotnet pack --no-build` takes ~0.5s on my machine.